mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -204,8 +204,8 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 		COMMENT Sets the current screen to a new screen.
 		COMMENT
 		COMMENT <p>If the screen being opened is {@code null} and the client is not in game, the title screen will be opened.
-		COMMENT If the currently opened screen is {@code null} and player is dead, then the death screen will be opened.
-		COMMENT Otherwise, the currently open screen will be closed.
+		COMMENT If the previous screen is {@code null} and the player is dead, then the death screen will be opened.
```

Comment:
So the death screen would override the provided parameter screen when the player is dead, I assume?

Reply:
No, see the line above - this only applies when the new screen is null.

Reply:
Hmm, so a table format would be better:
```
<p>If the new screen is {@code null}:
<ul>
<li>if the client is not in game, the title screen will be opened;</li>
<li>if the previous screen is {@code null} and the player is dead, then the death screen will be opened.</li>
</ul>
```